content/gotraining.slide

```diff
@@ -9,10 +9,229 @@ https://github.com/anz-bank/go-training
 
 * Introduction
 
-* Test
 
-# * Go history
 
-# * Why Go at ANZ
+* Agenda (11 - 15 February)
 
-# * Ping server Java Spring Boot vs Go
+- Introduction to Go
+- Go language fundamentals
+- Go standard library
+- Go tool chain
+- Third party Go libraries used at ANZ
+- Project work
+
+Find Slides at [[https://gotraining.appspot.com]] ([[https://github.com/anz-bank/go-slides][source]] on GitHub)
+
+
+* Day 1 - Schedule
+
+*Morning:* Introduction to Go
+
+- Overview and history
+- Demo
+
+*Afternoon:* Hello world and first PR
+
+- Interlude: towards continuous deployment (CD)
+- Go language fundamentals
+- First lab
+
+* History and Overview
+
+* History of Go
+
+.image img/xkcd_compiling.png
+.caption [[https://xkcd.com/303/]]
+
+2007: Robert Griesemer, Rob Pike, and Ken Thompson start working on go at Google
+2008: Russ Cox (go mod) and Ian Taylor (gcc frontend) join
+2009: Go goes Open Source
+2012: Go version 1.0
+
+
+* What is Go?
+
+- A simple language that is easy to learn and read
+- Statically typed, but with a dynamic feel
```

Comment:
maybe worth saying something about what kind of a static type system it has. i'm not quite sure what the right word is, perhaps something along the lines of "inexpressive" or "simple" -- like, there are many properties about a program that cannot be expressed in go's static type system, that can be expressed and checked in other type systems.   e.g. explicitly go type system lacks generics, although arguably that's not a type system thing, that's a code generation thing...

Reply:
Yeah, I think you have a point, and the "Structural typing" goes along with this too. 
I think I will leave this for the speaker notes though if you don't mind.